packages/rsbuild-plugin-angular/src/lib/ssr/server.ts

```diff
@@ -2,6 +2,7 @@ import { APP_BASE_HREF } from '@angular/common';
 import { CommonEngine } from '@angular/ssr/node';
 import express from 'express';
 import { dirname, join, resolve } from 'node:path';
+import { ApplicationRef, Type } from '@angular/core';
```

Comment:
If these are just type usages, we should change to

```suggestion
import type { ApplicationRef, Type } from '@angular/core';
```

And update the dependency in the package.json to `devDependency` with a range.
Pre-empting future backwards compat support.